<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>new_issue_form</class>
 <widget class="QDialog" name="new_issue_form">
  <property name="windowModality">
   <enum>Qt::ApplicationModal</enum>
  </property>
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1000</width>
    <height>800</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>New Issue Form</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <item>
    <widget class="QLabel" name="subject_label">
     <property name="text">
      <string>Subject:</string>
     </property>
     <property name="buddy">
      <cstring>subject_line_edit</cstring>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QLineEdit" name="subject_line_edit">
     <property name="toolTip">
      <string>Type a short description</string>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QLabel" name="description_label">
     <property name="text">
      <string>Description:</string>
     </property>
     <property name="buddy">
      <cstring>description_plain_text_edit</cstring>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QPlainTextEdit" name="description_plain_text_edit"/>
   </item>
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout_2">
     <item>
      <spacer name="horizontalSpacer">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>40</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
     <item>
      <widget class="QPushButton" name="cancel_button">
       <property name="text">
        <string>Cancel</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="ok_button">
       <property name="text">
        <string>Ok</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <tabstops>
  <tabstop>subject_line_edit</tabstop>
  <tabstop>description_plain_text_edit</tabstop>
  <tabstop>ok_button</tabstop>
 </tabstops>
 <resources/>
 <connections>
  <connection>
   <sender>cancel_button</sender>
   <signal>clicked()</signal>
   <receiver>new_issue_form</receiver>
   <slot>reject()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>851</x>
     <y>982</y>
    </hint>
    <hint type="destinationlabel">
     <x>790</x>
     <y>973</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
